libs/hdf-converters/test/mappers/forward/checklist_mapper.spec.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs';
```

Comment:
this is a regression test for the mapper

we should have regression tests for every middle format on our way to the mapper itself - ex. when going from xsd to jsonix type, is it the same?  if not, could be something wrong with jsonix, could be that the xml schema changed so we gotta rewrite everything on down.

additionally, we should add some sanity checks for making sure that the parsing fails when it receives an incorrect value

Reply:
Tests are updated